Tools/TestWebKitAPI/Tests/WebCore/DisplayListRecorderTests.cpp

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
Can't we just add a layout test similar to `displaylists/layer-dispay-list.html` which has the sequence you mentioned in the change log and verify the recorded display list will have `SetState` before `Save`?

Reply:
Well, those are good, except
- cannot target specifically all the c++ entry points that are possible
- cannot assert specifically any of the c++ entry points of the graphics context state

E.g. the idea of the test is that whatever the display list does, the corresponding non-display-list state must be the same as the state generated by the display list to the new graphics context.

Like, you cannot test this set of commands:
```
c.setShouldAntialias(false);
```